nautobot/extras/models/customfields.py

```diff
@@ -589,6 +585,16 @@ def to_form_field(
 
         return field
 
+    def to_filter_form_field(self, *args, **kwargs):
+        """Return a filter form field suitable for filtering a CustomField's value for an object."""
+        form_field = self.to_form_field(*args, **kwargs)
+        if self.type == CustomFieldTypeChoices.TYPE_SELECT:
+            # Remove the blank choice, as StaticSelect2Multiple do not need a blank choice
+            # Its blank by default
+            choices = form_field.choices[1:]
```

Comment:
Looks like this: https://github.com/nautobot/nautobot/actions/runs/4729451600/jobs/8392681922?pr=3574#step:7:161 is breaking a test case. In some instances, the first choice is not blank.

Reply:
I think they are only blank if the `add_blank_choice()`  is included in the choices call.

Reply:
The reason the test failed, is because CustomField Select type is no longer a normal select field, making the normal way of getting its value incorrect, but this has been fixed

Reply:
Sure seems like this should have the equivalent logic to the `required or default_choice is None` check that exists above?